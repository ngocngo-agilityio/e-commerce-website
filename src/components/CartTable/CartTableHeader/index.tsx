// Libs
import { memo } from 'react';
import { Th, Thead, Tr } from '@chakra-ui/react';

const CartTableHeader = () => {
  return (
    <Thead>
      <Tr>
        <Th></Th>
        <Th></Th>
        <Th>Product</Th>
        <Th>Price</Th>
        <Th>Quantity</Th>
        <Th>Total</Th>
      </Tr>
    </Thead>
  );
};

export default memo(CartTableHeader);
