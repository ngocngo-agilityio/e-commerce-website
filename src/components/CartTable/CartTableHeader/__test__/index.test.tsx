// Libs
import { Table } from '@chakra-ui/react';
import { render } from '@testing-library/react';

// Components
import CartTableHeader from '..';

describe('CartItem components', () => {
  test('should render CartTableHeader successfully', () => {
    const { container } = render(<CartTableHeader />, {
      wrapper: Table,
    });

    expect(container).toBeInTheDocument();
  });

  test('should match snapshot for CartTableHeader', () => {
    const { container } = render(<CartTableHeader />, {
      wrapper: Table,
    });

    expect(container).toMatchSnapshot();
  });
});
