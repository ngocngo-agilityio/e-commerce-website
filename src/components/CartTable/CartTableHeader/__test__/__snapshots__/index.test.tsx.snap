// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem components should match snapshot for CartTableHeader 1`] = `
<div>
  <table
    class="chakra-table css-0"
  >
    <thead
      class="css-0"
    >
      <tr
        class="css-0"
      >
        <th
          class="css-0"
        />
        <th
          class="css-0"
        />
        <th
          class="css-0"
        >
          Product
        </th>
        <th
          class="css-0"
        >
          Price
        </th>
        <th
          class="css-0"
        >
          Quantity
        </th>
        <th
          class="css-0"
        >
          Total
        </th>
      </tr>
    </thead>
  </table>
</div>
`;
